core/bin/external_node/src/main.rs

```diff
@@ -34,9 +35,14 @@ use zksync_dal::{healthcheck::ConnectionPoolHealthCheck, ConnectionPool};
 use zksync_health_check::CheckHealth;
 use zksync_state::PostgresStorageCaches;
 use zksync_storage::RocksDB;
+use zksync_types::ProtocolVersionId;
 use zksync_utils::wait_for_tasks::wait_for_tasks;
 
 mod config;
+mod metrics;
+
+const RELEASE_MANIFEST: &str =
+    std::include_str!("../../../../.github/release-please/manifest.json");
```

Comment:
This most probably is not included in the docker image, so should be included to work in the current state. Also doesn't look good to depend on some file in .github folder in the code, but I don't really have any alternatives

Reply:
`include_str` is a compile-time thing.

Reply:
This is ugly but this is because our versioning in Cargo.toml does not match the actual versions in that manifest, so unless we fix versioning this is going to be this way.